employees/common/exports.py

```diff
@@ -230,6 +232,24 @@ def _get_report_date_and_daily_hours(
         self._last_date = date
         return (report_date, daily_hours)
 
+    def _set_row_height(self, description: str) -> None:
+        # If the description has \ n or text exceeds 100 characters, it is automatically moved to the new line.
+
+        new_lines_in_description = description.count("\n")
+        splitted_description = description.split("\n")
+        rows_in_xlsx = 0
+
+        for row in splitted_description:
+            length_of_row = len(row)
+            rows_in_xlsx = int(length_of_row / 100)
```

Comment:
I don't get why you need iterate over all `splitted_description` to set this value, as it will have values set in last iteration of `spitted_description`. You can simply just select the last element with `splitted_description[-1]`.

Reply:
I iterate over all because over description column width is 100 chars and then We have wrapped text, so I have to check if there is no some rows with more than 100 chars. Because we could write reports like that:
```
First row
Second row
Third row
```
or just in one line without any enters.
``` 
First row Second row Third row
```
How will help me taking just last element ? I found that this is not working in all options so I fixed it up but there is again for and one more if condition

Reply:
Now after the fix it makes more sense. Previously the code was working like:
```
for x in y:
    a = something(x)
```

so the result was the same as `a = something(y[-1])`.

Reply:
Yes I know, I just realized this when I look for it this morning